electrumx/server/db.py

```diff
@@ -614,9 +613,8 @@ def flush_utxo_db(self, batch, flush_data: FlushData):
                 exponent = value[HASHX_LEN + SCRIPTHASH_LEN + 8: HASHX_LEN + SCRIPTHASH_LEN + 8 + 2]
                 tx_numb = value[-TXNUM_LEN:]  
                 batch_put(b'i' + location_key + atomical_id, hashX + scripthash + value_sats + exponent + tx_numb) 
```

Comment:
We need to ensure the `b'i'` index is written even if the utxo record is deleted between fast syncing.

If we do not use the "tombstone" pattern then what can happen is during a resync the UTXO is created, then spent before it is flushed to disk.  Then there will be no `b'i'` entry on disk for that utxo because it was  created and spent before it was persistent.

The reason we want the `b'i'` entry is that it provides a complete audit trail of the history of a utxo.